include/crow/websocket.h

```diff
@@ -94,6 +95,24 @@ namespace crow
                     return;
                 }
 
+                std::string requested_subprotocols_header = req.get_header_value("Sec-WebSocket-Protocol");
+                if (!subprotocols.empty() || !requested_subprotocols_header.empty())
+                {
+                    auto requested_subprotocols = utility::split(requested_subprotocols_header, ", ");
+                    auto subprotocol = std::find_first_of(subprotocols.begin(), subprotocols.end(), requested_subprotocols.begin(), requested_subprotocols.end());
```

Comment:
Afaict std::find_first_of is only available in C++17 or later. According to my knowledge crow should compile using C++11/14 as well.

Reply:
I will change it to be C++11/14, didn't see it was C++ 17